src/main/deploy/states/release.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<!DOCTYPE statedata SYSTEM "stateData.dtd">
+<statedata>
+	<controlData identifier="openloop" 
+	             mode="POSITION_DEGREES"/>
+	<mechanismTarget stateIdentifier="SERVO_OPEN_OPEN"
+	                 controlDataIdentifier="openloop"
+                     controlDataIdentifier2="openloop"
+					 value="0.0"
+                     secondValue="0.0"/>
+
+	<mechanismTarget stateIdentifier="SERVO_OPEN_CLOSED"
+	                 controlDataIdentifier="openloop"
+                     controlDataIdentifier2="openloop"
+					 value="0.0"
+                     secondValue="0.0"/>
+
+	<mechanismTarget stateIdentifier="SERVO_CLOSED_OPEN"
+	                 controlDataIdentifier="openloop"
+                     controlDataIdentifier2="openloop"
+					 value="0.0"
```

Comment:
value and secondValue should have different values for the open and closed states 

for instance, OPEN_OPEN could both be 0.0, but then OPEN_CLOSED might be 0.0 and 45.0